### PR TITLE
fix: fixed prob with avatar type bot and click

### PR DESCRIPTION
**Pull request by ionutrusu-bytex**
## Description

https://github.com/momentum-design/momentum-ui/issues/431
fixed prb with avatar type both and onclick set to true

## Screenshots:
**Before** (If applicable):

**After**:

image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by bfbiggs** [comment]
Were you able to look at the styles to see why this is happening?

File: react/src/lib/Avatar/index.js [hunk 1]
```diff
@@ -163,7 +163,7 @@ class Avatar extends React.Component {
       <div
         className={
           'md-avatar' +
-          `${(onClick && ` md-avatar--clickable`) || ''}` +
+          `${((onClick && type!=='bot') && ` md-avatar--clickable`) || ''}` +
```
Comment (bfbiggs) [inline react/src/lib/Avatar/index.js:166]: If the style isn't fixable then let's remove the extra parens here and add spaces around the '!=='

Comment (ionutrusu-bytex) [reply react/src/lib/Avatar/index.js]: @bfbiggs fixed

**Comment by ionutrusu-bytex** [comment]
@holsted 
image bots are still clickable, obs how i pass onClick prop true and type bot...on click u can see the alert window

**Comment by bfbiggs** [comment]
> Sorry, why are bots not clickable?

@holsted yeah I took a look - looks like both bot style and clickable style both modify the :after pseudo element. As long as the lack of the clickable styles aren't required by design then I think this fix is sufficient.  My question is instead of this patch - should we actually look to move bot to function the same way as notification badge and failure badge. This is the only type attribute that modifies the :after element like this.

`     {failureBadge && <span className='md-avatar__failure-badge' />}  
{hasNotification && <span className='md-avatar__notification-badge' />}`

**Comment by aimeex3** [comment]
> My question is instead of this patch - should we actually look to move bot to function the same way as notification badge and failure badge. This is the only type attribute that modifies the :after element like this.

agree with @bfbiggs. that way the onclick behavior still works correctly for bot avatars

**Comment by ionutrusu-bytex** [comment]
@bfbiggs is this PR still valid?

**Pull request closed without merging**